### DISCUSSION TITLE
Show an indication when the suggested price is actually the lowest one

### DISCUSSION THREAD

**Discussion by nikoskleidis** [keybraker/reSkroutzed#11] [Ideas]
Great job on this extension it is actually very useful and can inform us about the dark patterns that are being used!

This image is now shown on all pages without calculating anything.
![image](https://user-images.githubusercontent.com/4397396/236168171-d7e54267-7128-400f-9ac6-a20987cdc975.png)

My suggestion is to show it only in cases there is a lower price down below (and maybe say what this price is)
and hide it or display a green indication that everything looks good when the price is actually the lowest one found.

**Comment by keybraker** [comment]
Good recommendation @nikoskleidis ! Will be taken into consideration on the next release.

**Reply by gtopsis** [reply]
Some thoughts.
Note: I'm not a frequent user of platform skroutz, so be aware that some thoughts may be irrelevant.

1. I'm wondering if this functionality is out of scope of this extension's goal. I'm not sure if the warning text above is associated with a sponsored store/product. If so, then cool.
2. It seems to me more like a "patch" on fixing their dark design pattern on promoting their feature "Buy via Skroutz" rather than the best choice for the client. Like admitting that the previous behavior was misleading for the clients and decided to add an extra message <<Maybe the following price is not the lowest>>.
3. In order someone to be able to claim that the promoted price **is indeed the lowest one** (with a green color) as suggested, someone may need to fetch all products from all pages of results, As far as I know this extension is scrapping the current page of results each time, therefore it has no idea what other products exist.

**Reply by keybraker** [reply]
1. This is as you mention a dark pattern and in continuation a promotion for their own product (and the sponsored stores who pay for this kind of promotion). Skroutz, as the market leader, promotes itself and not fairly displaying the lowest price it was designed to do. This is a big and open question though on how for example a store owner places their products in their store.
2. This is a dark pattern and one that directly correlates to the rest.
3. This flag exists only in the product page and not the category page, when you load a product, you load the entire list of stores listed in Skroutz. Just to add on that, if you click Buy through Skroutz on the category page, you are actively seeking it, but in the case of the product page you are bombarded by promotional and sponsored material before you reach the actual list.

**Reply by gtopsis** [reply]
Thanks. Clearer to me.
About (1), which for me is a good question. 

For me, visually degrading **on purpose** the info that a product is sponsored, although you sort products by price (or other criteria), is a dark design pattern. Include and show sponsored products in a platform is not. It is not my flavor and that's why I will choose another similar platform closer to my ethical axis.

For me, if skroutz sorts all products by the same criteria and for those which are sponsored it adds a **highlighted** label Sponsored then it's fine.

**Reply by keybraker** [reply]
I agree and the thing is, that they don't. And this is the reason I think this needs a fix as it flags a dark pattern.
